build.go

```diff
@@ -13,17 +14,26 @@ func Build(logger scribe.Logger) packit.BuildFunc {
 		logger.Title("%s %s", context.BuildpackInfo.Name, context.BuildpackInfo.Version)
 
 		configParser := NewRuntimeConfigParser()
-		config, err := configParser.Parse(filepath.Join(context.WorkingDir, "*.runtimeconfig.json"))
+		var command string
```

Comment:
Does this need to be declared as a variable explicitly? If it does could me move it closer to where it is initially set?

Reply:
oops that's left over from a previous implementation of this code. Good catch